mne_bids_pipeline/_main.py

```diff
@@ -60,10 +68,40 @@ def main():
         '--no-cache', dest='no_cache', action='store_true',
         help='Disable caching of intermediate results.')
     options, args = parser.parse_args()
+
+    if options.create_config is not None:
+        config_target_path = pathlib.Path(options.create_config)
+        config_source_path = pathlib.Path(__file__).parent / '_config.py'
+        if config_target_path.exists():
+            raise FileExistsError(
+                f'The specified path already exists: {config_target_path}'
+            )
+
+        # Create a template by commenting out most of the lines in _config.py
+        config: List[str] = []
+        with open(config_source_path, 'r', encoding='utf-8') as f:
+            for line in f:
+                line = (
+                    line if line.startswith(('#', '\n', 'import', 'from'))
+                    else f'# {line}'
+                )
+                config.append(line)
+
+        config_target_path.write_text(''.join(config), encoding='utf-8')
```

Comment:
Can you put this in a helper function in `mne_bids_pipeline/_*.py` somewhere? It helps keep `_main` just about triaging args and control flow, especially once we decide to make this more sophisticated...

Reply:
Done